CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.18.0
+
+* Require Dart 3.0.
+* Mark mixin classes as `mixin class`.
+
 ## 1.17.2
```

Comment:
This version was never published, so we can probably just drop the changelog entry.